src/modules/OTA-api/data-components/Contributors.tsx

```diff
@@ -1,91 +1,189 @@
+    type: 'core',
+    avatar_url: 'https://avatars.githubusercontent.com/u/1098708?v=4',
+    html_url: 'https://github.com/Ndpnt',
+  },
+  {
+    login: 'Martin Ratinaud',
+    type: 'core',
+    avatar_url: 'https://avatars.githubusercontent.com/u/4191809?v=4',
+    html_url: 'https://github.com/martinratinaud',
+  },
+  {
+    login: 'Clément Biron',
+    type: 'core',
+    avatar_url: 'https://avatars.githubusercontent.com/u/364319?v=4',
+    html_url: 'https://github.com/clementbiron',
+  },
+  {
+    login: 'Matti Schneider',
+    type: 'core',
+    avatar_url: 'https://avatars.githubusercontent.com/u/222463?v=4',
+    html_url: 'https://github.com/MattiSG',
+  },
+  {
+    login: 'Adrien Fines',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/41912392?v=4',
+    html_url: 'https://github.com/AdrienFines',
+  },
+  {
+    login: 'Lucas Verney',
+    type: 'contributors',
+    avtar_url: 'https://avatars.githubusercontent.com/u/58298410?v=4',
+    html_url: 'https://github.com/LVerneyPEReN',
+  },
+  {
+    login: 'Vincent Viers',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/30295971?v=4',
+    html_url: 'https://github.com/vviers',
+  },
+  {
+    login: 'Tom Houriez',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/70654947?v=4',
+    html_url: 'https://github.com/THouriezPEReN',
+  },
+  {
+    login: 'Aaronj Sugarman',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/82889095?v=4',
+    html_url: 'https://github.com/AaronjSugarman',
+  },
+  {
+    login: 'GatienH',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/6501059?v=4',
+    html_url: 'https://github.com/GatienH',
+  },
+  {
+    login: 'Michiel de Jong',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/408412?v=4',
+    html_url: 'https://github.com/michielbdejong',
+  },
+  {
+    login: 'Siegrid Henry',
+    type: 'alumnis',
+    avatar_url: 'https://avatars.githubusercontent.com/u/49791551?v=4',
+    html_url: 'https://github.com/SiegridHenry',
+  },
+  {
+    login: 'Chirsitan Quest',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/1202668?v=4',
+    html_url: 'https://github.com/cquest',
+  },
+  {
+    login: 'Marius Karnauskas',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/1094012?v=4',
+    html_url: 'https://github.com/karnauskas',
+  },
+  {
+    login: 'TOSDR',
+    type: 'contributors',
+    avatar_url: 'https://avatars.githubusercontent.com/u/3402969?s=200&v=4',
+    html_url: 'https://github.com/tosdr',
   },
 ];
 
-const Contributors: React.FC<ContributorsProps> = React.memo(({ subtitle }) => {
-  const { data } = useSWR('/api/ota/contributors/all', {
-    revalidateOnMount: true,
+const getContributorsByType = (type: ContributorsProps['type']) => {
```

Comment:
I think it would be simpler and more efficient to maintain three different arrays of contributors rather than adding a property to all contributors and then loop over them three times on every page load 😉